src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/index.e2e.tsx

```diff
@@ -26,11 +27,26 @@ function ComposerWithSuggestionsE2e(props: ComposerWithSuggestionsProps, ref: Fo
 
         // We need to wait for the component to be mounted before focusing
         setTimeout(() => {
-            if (!(ref && 'current' in ref)) {
-                return;
-            }
+            const setFocus = () => {
+                if (!(ref && 'current' in ref)) {
+                    return;
+                }
 
-            ref.current?.focus(true);
+                ref.current?.focus(true);
+
+                setTimeout(() => {
+                    // and actually let's verify that the keyboard is visible
+                    if (Keyboard.isVisible()) {
+                        return
+                    }
+
+                    ref.current?.blur();
+                    setFocus();
+                    // 500ms is enough time for any keyboard to open
+                }, 500);
+            };
+
+            setFocus();
```

Comment:
If this is a fix for an existing bug, does it mean that we need to increase the timeout that wraps it (currently 1ms)? 
Could there be a cycling issue here?
I understand that this is for testing and wouldn't affect users, so this is just out of curiosity

Reply:
> If this is a fix for an existing bug, does it mean that we need to increase the timeout that wraps it (currently 1ms)?

No, we don't need. Sometimes it may work within 1ms, sometimes not - and in this case we give 500ms to be sure keyboard is opened (otherwise we blur input, set focus again and repeat the action).

> Could there be a cycling issue here?

I did many tests and haven't discovered ant cycling here 👀